present/trace.go

```diff
@@ -72,14 +74,24 @@ func SpansToSVG(w io.Writer, spans []*collect.FinishedSpan) error {
 	}
 
 	for id, s := range spans {
+		color := "rgb(128,128,255)"
+		switch {
+		case s.Panicked:
+			color = "rgb(255,0,0)"
+		case unwrapError(s.Err) == context.Canceled:
```

Comment:
i think there's a variable defined for contextCanceled that's go-release-specific somewhere maybe? maybe that's some other thing we've open sourced.